Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py

```diff
@@ -1418,94 +1799,135 @@ def get_file_analysis_result(
     """
     Get the report of file based on report id and sends the result to demist war room
     :type client: ``Client``
-    :param client: client object to use http_request.
+    :param client: client object used to initialize pytmv1 client.
     :type args: ``dict``
     :param args: args object to fetch the argument data.
     :return: sends data to demisto war room.
     :rtype: ``dict`
     """
-    report_id = args.get(REPORT_ID)
-    response = client.http_request(GET, GET_FILE_RESULT.format(reportId=report_id))
-    risk = response.get("riskLevel", "")
-    risk_score = client.incident_severity_to_dbot_score(risk)
-    sha256 = response.get("digest", {}).get("sha256")
-    md5 = response.get("digest", {}).get("md5")
-    sha1 = response.get("digest", {}).get("sha1")
-    reliability = demisto.params().get("integrationReliability")
+    # Required Params
+    report_id = args.get(REPORT_ID, EMPTY_STRING)
+    # Optional Params
+    poll = args.get(POLL, TRUE)
```

Comment:
as the poll expected to be a `bool` val - `'false'` will considered as `True` as in python `bool('false') 
== bool('False') == True`
```suggestion
    poll = argToBoolean(args.get(POLL, TRUE))
```

please fix it in **all the places** that you expected to get `bool` value

Reply:
corrected.